### PR TITLE
cleanup some unused stuff

### PR DESCRIPTION
**Pull request by laurelfan**
- clean up some unused routes
- move some redirects out of controllers and into routes.rb
- delete some unused views and controller actions

### CODE REVIEW

File: dashboard/test/integration/redirects_test.rb [hunk 1]
```diff
@@ -59,4 +59,14 @@ class RedirectsTest < ActionDispatch::IntegrationTest
     assert_redirected_to '/c/1/generate_image'
   end
 
+  test "old teacher dashboard redirects to new teacher dashboard" do
+    urls = %w{/followers /followers/manage /followers/sections /stats/students /sections/new /sections/1/edit}
+
+    urls.each do |url|
+      get url
+      assert_response :redirect#, "for url #{url}"
+      assert_redirected_to 'http://localhost.code.org:3000/teacher-dashboard', "for url #{url}"
```
Comment (joshlory) [inline dashboard/test/integration/redirects_test.rb]: Looks like this URL needs to be generalized (Travis CI error)

**Comment by laurelfan** [comment]
fixed the tests... can you review?

File: dashboard/config/routes.rb [hunk 2]
```diff
@@ -83,16 +86,10 @@
     post 'clone', to: 'levels#clone'
   end
 
-  resources :games
-
-  get 'builder', to: 'levels#builder'
-  post 'upload_maze_level', to: 'levels#upload_maze_level'
   post 'create_custom', to: 'levels#create_custom'
```
Comment (joshlory) [inline dashboard/config/routes.rb]: Looks like this one can be removed too.

**Comment by joshlory** [comment]
Looks good!  Glad to see some of this old code cleaned out :+1:

File: dashboard/config/routes.rb [hunk 3]
```diff
@@ -83,16 +86,10 @@
     post 'clone', to: 'levels#clone'
   end
 
-  resources :games
```
Comment (joshlory) [inline dashboard/config/routes.rb]: Thanks for catching this & :activities above

**Pull request merged**